features/admin.users.v1/pages/users.tsx

```diff
@@ -221,6 +225,32 @@ const UsersPage: FunctionComponent<UsersPageInterface> = (
         !isSubOrganization()
     );
 
+    // Combine the search query and account status filters query.
+    const combinedUserFilter: string = useMemo(() => {
```

Comment:
in the user-list.tsx use the combinedUserFilter instead of searchQuery to display the page when no results are found

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/8171/commits/a35f3f308db99f0f624552fce3446dbdefa8b471